tests/test_client_functional.py

```diff
@@ -632,6 +632,28 @@ async def handler(request: web.Request) -> web.Response:
     assert txt == "Test message"
 
 
+async def test_ssl_client_alpn(
+    aiohttp_server: AiohttpServer,
+    aiohttp_client: AiohttpClient,
+    ssl_ctx: ssl.SSLContext,
+) -> None:
+
+    async def handler(request: web.Request) -> web.Response:
+        sslobj = request.transport.get_extra_info("ssl_object")
+        assert sslobj.selected_alpn_protocol() == "http/1.1"
+        return web.Response(text="Test message")
```

Comment:
Might make sense to return the protocol in the response and then assert it below?